internal/pkg/deploy/cloudformation/stack/backend_svc.go

```diff
@@ -61,6 +62,17 @@ func NewBackendService(mft *manifest.BackendService, env, app string, rc Runtime
 
 // Template returns the CloudFormation template for the backend service.
 func (s *BackendService) Template() (string, error) {
+	desiredCountLambda, err := s.parser.Read(desiredCountGeneratorPath)
+	if err != nil {
+		return "", fmt.Errorf("read desired count lambda: %w", err)
+	}
+	if s.updateID == "" {
+		id, err := randomUpdateID()
+		if err != nil {
+			return "", err
+		}
+		s.updateID = id
```

Comment:
Do we need to store this part of the state or can we just generate a random UUID everytime it's called?

Reply:
we just generate a random one every time it is called. The member variable is just for testing